subsys/bluetooth/audio/Kconfig.bap

```diff
@@ -70,13 +62,19 @@ config BT_AUDIO_CODEC_CFG_MAX_METADATA_COUNT
 	  This option defines the maximum number of LTV entries a metadata can
 	  store.
 
-config BT_AUDIO_CODEC_CAP_MAX_METADATA_COUNT
-	int "Codec Capabilities Metadata Count"
-	default 2
-	range 1 128
+config BT_AUDIO_CODEC_CAP_MAX_DATA_SIZE
```

Comment:
Should the CAP related configs be in Kconfig.cap?

Reply:
Hmm, `CAP` here is for `capabilities` (similar to `codec_cap`, but I certainly see where this may be confusing. We could use `capa` for `capabilities` or maybe `caps`?

Reply:
> Hmm, `CAP` here is for `capabilities` (similar to `codec_cap`, but I certainly see where this may be confusing. We could use `capa` for `capabilities` or maybe `caps`?

HA, you are right :) sorry... and yes, maybe caps could be better as I hit the quick confusion when not reading the text

Reply:
> maybe caps could be better as I hit the quick confusion when not reading the text

I'd like to get some input from at least 1 more whether this is confusing or not, before I do a large renaming again :D 

@MariuszSkamra @fredrikdanebjer @alexsven @asbjornsabo 
Do you think `CODEC_CAP`/`codec_cap` can be confused with something relating to the common audio profile (CAP)?

Reply:
Yes, it can be confusing for sure haha. I think we had this exact issue some months back in the pacs implementation as well.

Sadly, I dont know if there is a good abbreviation for capabilities. Did you consider to now abbreviate it at all in this case?

Reply:
> Sadly, I dont know if there is a good abbreviation for capabilities. Did you consider to not abbreviate it at all in this case?

Hmm, `capability` is just quite a long word :D 
So it would be `bt_audio_codec_capability` and `CONFIG_BT_AUDIO_CODEC_CAPABILITY_MAX_DATA_SIZE`, which perhaps isn't too bad, but it does get long when you then do something like 
```c
static const struct bt_audio_codec_capability codec_capability = BT_AUDIO_CODEC_CAPABILITY_LC3(...);
```